cli/defenseclaw/file_permissions.py

```
@@ -992,7 +1046,16 @@ def atomic_write_private(
             replace_file_durable(tmp, target)
             tmp = ""
             if os.name == "nt":
-                _verify_or_repair_windows_private_target(target)
+                if windows_managed_custody:
+                    from defenseclaw import windows_acl
+
+                    if windows_acl.capture_path(target) != managed_security:
+                        raise OSError("managed Windows security changed during atomic publication")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)